ci-operator/step-registry/ipi/conf/vsphere/zones/ipi-conf-vsphere-zones-commands.sh

```diff
@@ -24,6 +24,8 @@ mapfile -t vips < "${SHARED_DIR}/vips.txt"
 CONFIG="${SHARED_DIR}/install-config.yaml"
 base_domain=$(<"${SHARED_DIR}"/basedomain.txt)
 machine_cidr=$(<"${SHARED_DIR}"/machinecidr.txt)
+ova_url=$(openshift-install coreos print-stream-json | jq -r '.architectures.x86_64.artifacts.vmware.formats.ova.disk.location')
```

Comment:
It seems that we override the default behavior if appending field `template` directly in this zone configure step? Is it on purpose? 

And this step should be available on 4.12+, but field `template` starts to be supported on 4.14, so job might be failed on old version?